src/search-manager/ClearFiltersButton.tsx

```diff
@@ -4,19 +4,34 @@ import { Button } from '@openedx/paragon';
 import messages from './messages';
 import { useSearchContext } from './SearchManager';
 
+type ClearFiltersButtonProps = {
+  variant?: 'link' | 'primary',
+  size?: 'sm' | 'md' | 'lg' | 'inline',
+};
+
+const defaultProps : ClearFiltersButtonProps = {
+  variant: 'link',
+  size: 'sm',
+};
+
```

Comment:
We are removing propTypes when moving to typescript. For now, we still need the `eslint-disable`

```suggestion
/* eslint-disable react/require-default-props */
type ClearFiltersButtonProps = {
  variant?: 'link' | 'primary',
  size?: 'sm' | 'md' | 'lg' | 'inline',
};

```

Reply:
Yup. Note that I have a proposal for disabling this `default-props` rule: https://github.com/openedx/eslint-config/pull/156

Reply:
Ah thank you for that tip @rpenido ! Fixed with [74c2d42](https://github.com/openedx/frontend-app-course-authoring/pull/1147/commits/74c2d42bdd708e1a0c8004a09e0896b628236088).